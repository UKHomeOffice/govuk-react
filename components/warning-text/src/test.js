import React from 'react';
import ReactDOM from 'react-dom';
import WarningText from './';

describe(WarningText, () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<WarningText>Example</WarningText>, div);
  });
});
