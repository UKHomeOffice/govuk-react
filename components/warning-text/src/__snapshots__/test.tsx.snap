// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WarningText to match snapshot 1`] = `
<WarningTextWithKnobs>
  <WarningText>
    <src__StyledWarningText>
      <div
        className="src__StyledWarningText-sc-14ghfbq-0 igQrNb"
      >
        <src__IconImportantWrapper>
          <div
            className="src__IconImportantWrapper-sc-14ghfbq-1 kMdMZl"
          >
            <IconImportant
              fill="currentColor"
              title="icon important"
            >
              <SVG
                fill="currentColor"
                preserveAspectRatio="xMidYMid meet"
                title="icon important"
                viewBox="0 0 35.000000 35.000000"
              >
                <svg
                  fill="currentColor"
                  height="100%"
                  preserveAspectRatio="xMidYMid meet"
                  style={
                    Object {
                      "display": "block",
                    }
                  }
                  version="1.1"
                  viewBox="0 0 35.000000 35.000000"
                  width="100%"
                >
                  <title>
                    icon important
                  </title>
                  <g
                    transform="translate(0.000000,35.000000) scale(0.100000,-0.100000)"
                  >
                    <path
                      d="M100 332 c-87 -48 -125 -155 -82 -232 48 -87 155 -125 232 -82 87 48
125 155 82 232 -48 87 -155 125 -232 82z m100 -122 c0 -53 -2 -60 -20 -60 -18
0 -20 7 -20 60 0 53 2 60 20 60 18 0 20 -7 20 -60z m0 -111 c0 -12 -7 -19 -20
-19 -19 0 -28 28 -14 43 11 11 34 -5 34 -24z"
                    />
                  </g>
                </svg>
              </SVG>
            </IconImportant>
          </div>
        </src__IconImportantWrapper>
        <src__WarningTextWrapper>
          <strong
            className="src__WarningTextWrapper-sc-14ghfbq-2 bLQRYB"
          >
            A very long warning message. This includes a lot of text to give a good representation of a more average length warning. That way you can see more than one line wrapping.
          </strong>
        </src__WarningTextWrapper>
      </div>
    </src__StyledWarningText>
  </WarningText>
</WarningTextWithKnobs>
`;
